server/services/event.ts

```
@@ -385,13 +385,13 @@ export async function getEvents(oldEvents: boolean = false) {
 
 const allActiveEventsCache:
   | {
-      generatedAt: Date;
-      data: EventsFromDb;
-    }
+    generatedAt: Date;
```

ESLint

[error] Insert `··`

(prettier/prettier)